### ISSUE TITLE
Trying to run Chia Blockchain on MacOS 10.14 using release 1.0.1 and 1.0.2.  Stuck at Connecting to Wallet

### ISSUE THREAD

**Issue by cvee72** [Chia-Network/chia-blockchain#1609]
Trying to run Chia Blockchain on MacOS 10.14 using release 1.0.1 and 1.0.2.  Stuck at Connecting to Wallet

Tried deleting .chia directory and restarting macOS.
Same issue.
Can't even get beyond point to obtain key etc.

Do not see any log files in the .chia/mainnet/log directory.

Not sure how to proceed.

I do run a hackintosh.... unsure if that is related or not.

**Comment by iToads14** [comment]
I am having the same issue. Stuck at connecting to wallet.

Running Chia Blockchain on MacOS Catalina using 1.0.2 and 1.0.3.

**Comment by cvee72** [comment]
> I am having the same issue. Stuck at connecting to wallet.
> 
> Running Chia Blockchain on MacOS Catalina using 1.0.2 and 1.0.3.

I went onto Keybase and they helped me resolve the issue.
Turns out my issue was a hosts file problem.
I had some crazy entries which i removed.... and after reboot.. voila allowed me to generate a key.

Prior to that, they had me do :- 
cd /Applications/Chia.app/Contents/Resources/app.asar.unpacked/daemon
./chia init

and

./chia keys generate


Depending on the errors/messages that come out of those commands i guess directs you how to proceed.

Get on keybase and get support from the chia guys.
